skabase/SKALogger/SKALogger/release.py

```diff
@@ -13,6 +15,6 @@
 description = """A generic base device for Logging for SKA."""
 author = "cam"
 author_email = "cam at ska.ac.za"
-license = """BSD-3-Clause"""
+license = """none"""
```

Comment:
The license value is BSD-3-Clause.

Reply:
Yes, this happens everytime we recreate from Pogo. Will change it.